### DISCUSSION TITLE
REACTOR MELDOWN WARNING SYSTEM !!!!!!!!!!!!

### DISCUSSION THREAD

**Discussion by ArtemK216** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1421] [Q&A]
Ok so maybe im stupid and don't know how some things about this mod but I tried everything and I was unable to set up a system that WOULD WARN ME THAT MY REACTOR IS ABOUT TO EXPLODE, LIKE A THING THAT SHOULD HAVE BEEN SINCE DAY ONE IN THIS MOD,  IDK HOW HARD IS ITS TO SET UP WERE A REACTOR OR ITS CONTROL BLOCK EMITTS A REDSTONE SIGNAL WHEN IT GETS TO HOT SO I CAN PUT A SIREN AND IT TELLS ME OH SNAP THINGS ARE ABOUT TO GO SOUTH AND PREVENT IT, IT IS A THING WITH "REACTOR CONTROL BLOCK" AND "RESEARCH REACTOR", WHY IS IT NOT A THING WITH RBMK AND ZIRNOX ???? THATS MY ONLY QUESTION WHY ?! IS IT NOT A THING WWWWWWWWWWWWHHHHHHHHHHHHHHHHYYYYYYYYYYYYY!!!!!???? I GUESS IT IS THAT HARD TO MAKE !

**Comment by Vaern** [comment]
please use your indoor voice

**Reply by ArtemK216** [reply]
Well Ill see how you will be using your indoor voice when you loose a cople reactrors in survival

**Reply by Vaern** [reply]
why don't you test them in a creative world beforehand, and make sure they're always loaded in your survival world?

**Reply by ArtemK216** [reply]
I do that and everything works as intended but unforceen things happen and I want to know if my reactor is about to melt down I don't care what they add in the next version of the mod ADD REACTOR WARNING SYSTEM FIRST PLS !

**Reply by Vaern** [reply]
too bad nerd

**Reply by ArtemK216** [reply]
Bet that's all you do all day, join someone's discussion without saying anything useful and calling people nerds

**Reply by Vaern** [reply]
well, there's a variety of things you could do. "unforeseen consequences" are typically pretty easy to predict, you also need to chunkload your coolant loop for both reactors (only the PWR has some form of unload protection)

you could also use opencomputers, we have compatibility that allows you to interface with the reactors to do the exact thing you're asking for. i don't really get your aggression though, meltdowns will happen regardless, and i reserve the right to call you a nerd because i added the zirnox reactor

**Reply by ArtemK216** [reply]
There see a sensible answer,  I did't think about opencomputers I will give it a try, as for calling me nerds I guess you are one as well so ill give you that.
